AstSemantics.md

```diff
@@ -350,7 +352,7 @@ The same operations are available on 64-bit integers as the ones available for
 Floating point arithmetic follows the IEEE-754 standard, except that:
  - The sign bit and significand bit pattern of any NaN value returned from a
    floating point arithmetic operation other than `neg`, `abs`, and `copysign`
-   are computed nondeterministically. In particular, the "NaN propagation"
+   are not specified. In particular, the "NaN propagation"
```

Comment:
To be clear, even the same operation, with the same operand values, can return different values each time it is executed. But I see what you mean about the existing text being prescriptive. I'll be ok with "not specified" for now.